torch/_dynamo/variables/builder.py

```diff
@@ -512,6 +513,9 @@ def build_key_value(i, k, v):
             # along with other builtin debugging functions
             self.install_guards(GuardBuilder.BUILTIN_MATCH)
             return DebuggingVariable(value, source=self.source)
+        elif isinstance(value, logging.Logger):
```

Comment:
Do you know if this can support huggingface's custom logger?

Reply:
as it depends on `import transformers` I am doing on a separate class with try-except